src/activities/components/ActivityList.vue

```diff
@@ -82,16 +82,28 @@
       :offset="100"
       @load="displayMoreActivities"
     >
-      <ActivityItem
-        v-for="activity in displayedActivities"
-        :key="activity.id"
-        v-measure
-        :activity="activity"
-        :place-link="placeLink"
-        @join="$emit('join', arguments[0])"
-        @leave="$emit('leave', arguments[0])"
-        @detail="$emit('detail', arguments[0])"
-      />
+      <template
+        v-for="(day, index) in displayedActivitiesGroupedByDate"
+      >
+        <div
+          v-if="!dense"
+          :key="`day-${index}`"
```

Comment:
Minor: I think `day.date` would make a better `key`, as it's related to the data, not just the position in the array (we live update a lot of stuff via websockets, so the contents can change - although not sure about this particular case).

Reply:
see comment above...